modules/diary.py

```diff
@@ -247,14 +303,16 @@ def notes():
             contents = yaml.load(todays_notes_entry)
 
             click.echo('Today\'s notes:')
-            click.echo('----------------')
-            click.echo("  Time  | Note")
-            click.echo("--------|-----")
+            click.echo('---------------------')
+            click.echo("S.No. | Time   | Note")
+            click.echo("------|--------|-----")
 
+            counter=0;
             for entry in contents['entries']:
+                counter+=1
                 time = entry['time']
                 text = entry['text']
-                click.echo(time + "| " + text)
+                click.echo("--" + str(counter) + "---|" + time + "| " + text)
 
```

Comment:
please reformat the files you've changed using `autopep8` after you're finished working on them